<launch>
  <arg name="pkg_name" value="virtual_mirror_amadoa"/>
  <arg name="node_name" default="virtual_mirror_amadoa_node"/>
  <arg name="veh" doc="Name of vehicle. ex: megaman"/>
  <arg name="local" default="true" doc="true for running everything you possibly can on laptop. false for everything running on vehicle."/>
  <arg name="config" default="baseline" doc="Specify a config."/>
  <arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman." />
  
  <arg name="flip_direction" doc="horz or vert"/>
	
  <group ns="$(arg veh)">
    <node name="$(arg node_name)" pkg="virtual_mirror_amadoa" type="virtual_mirror_amadoa_node.py" output="screen">
    
      <rosparam command="load" file="$(find virtual_mirror_amadoa)/config/$(arg config)/$(arg pkg_name)/$(arg node_name)/$(arg param_file_name).yaml"/>

      <param name="~flip_direction" value="$(arg flip_direction)"/>

	    <!-- Subscribe Topics -->
	    <!-- <remap from="duckiebot_image" to="camera_node/image/compressed"/> -->

	    <!-- Publish Topics -->
	    <!-- <remap from="mirrored_image" to="virtual_mirror_amadoa_node/image/compressed"/> -->
	    <!-- <remap from="flip" to="virtual_mirror_amadoa_node/flip"/> -->
	  
	</node>
  </group>
	
</launch>
